server/src/models/user.model.ts

```diff
@@ -0,0 +1,85 @@
+import {Entity, model, property, belongsTo} from '@loopback/repository';
+import {Customer} from './customer.model';
+import {Role} from './role.model';
+
+@model({settings: {strict: true}})
+export class User extends Entity {
+  @property({
+    type: 'string',
+    id: true,
+    defaultFn: 'uuidv4',
+  })
+  id?: string;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  first_name: string;
+
+  @property({
+    type: 'string',
+    default: '',
+  })
+  middle_name?: string;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  last_name: string;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  email: string;
+
+  @property({
+    type: 'number',
+    required: true,
+    postgresql: {
+      dataType: 'bigint',
+    },
+  })
+  phone_number: number;
+
+  @property({
+    type: 'string',
+    required: true,
+  })
+  address: string;
+
+  @property({
+    type: 'date',
+    default: () => new Date(),
+  })
+  createdAt?: Date;
+
+  @property({
+    type: 'date',
+    default: () => new Date(),
+  })
+  updatedAt?: Date;
+
+  @belongsTo(() => Customer)
+  customerId: string;
+
+  @belongsTo(() => Role)
+  roleId: string;
+  // Define well-known properties here
+
+  // Indexer property to allow additional data
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  [prop: string]: any;
+
```

Comment:
these can be removed, at other places also

Reply:
I have removed it from all model